test/it/EnterpriseAcceptanceSpec.scala

```diff
@@ -0,0 +1,59 @@
+import java.time.Month.MARCH
+
+import play.api.http.ContentTypes.JSON
+import play.api.http.Status.OK
+import org.scalatest.OptionValues
+
+import uk.gov.ons.sbr.models.Period
+import uk.gov.ons.sbr.models.enterprise.{ Enterprise, Ern }
+
+import fixture.ServerAcceptanceSpec
+import it.fixture.ReadsEnterpriseUnit.enterpriseReads
+import repository.hbase.unit.enterprise.EnterpriseUnitColumns._
+import repository.hbase.unit.enterprise.EnterpriseUnitRowKey
+import support.WithWireMockHBase
+import support.sample.SampleEnterpriseUnit
+
+class EnterpriseAcceptanceSpec extends ServerAcceptanceSpec with WithWireMockHBase with OptionValues with SampleEnterpriseUnit {
+
+  private val TargetErn = Ern("1000000012")
+  private val TargetPeriod = Period.fromYearMonth(2018, MARCH)
+
+  private val EnterpriseUnitSingleMatchHBaseResponseBody =
+    s"""{"Row": ${
+      List(
+        aRowWith(key = s"${EnterpriseUnitRowKey(TargetErn, TargetPeriod)}", columns =
+          aColumnWith(name = ern, value = TargetErn.value),
+          aColumnWith(name = entref, value = SampleEnterpriseReference),
+          aColumnWith(name = name, value = SampleEnterpriseName),
+          aColumnWith(name = postcode, value = SamplePostcode),
+          aColumnWith(name = legalStatus, value = SampleLegalStatus),
+          aColumnWith(name = employees, value = SampleNumberOfEmployees.toString),
```

Comment:
We cannot have an acceptance test for every possible scenario, that way leads to incredibly slow builds.  However, nor is the acceptance test fixed in stone.  As your implementation develops and you discover more of the edge cases, see if some of them can be reflected in the test case.

Once you are happy you can support all fields, you may therefore want to have one of the optional fields as missing.  Of course, you have unit tests that prove you can handle all the fields correctly - but such a change helps the acceptance test reflect more of the "real world" case - rather than a perfect scenario where every single field is populated.